### DISCUSSION TITLE
Remove the "Playing" status before the custom status

### DISCUSSION THREAD

**Discussion by Fusity** [Rapptz/discord.py#6773] [Q&A] [answered]
I want to remove the "Playing" status when I write a custom status like this :
here my  customs status`status = [f"{config.prefix}help",
        "A votre service",
        "Powered by ender_creeps and Fusity.tar.gz",
        "regarder les demandes de la PacifiqueCreation"
        ]`
        
And the code to add the status :
`@tasks.loop(seconds = 5)
async def changeStatus():
    game = discord.Game(random.choice(status))
    await bot.change_presence(status = discord.Status.online, activity = game)`

But the status on my discord bot sayq "Playing A votre service"
How can i remove it ?

**Comment by AbstractUmbra** [comment, marked as answer]
You cannot remove this, no. Discord bots cannot have custom activities like members can.

You can have either: `Playing ...`, `Watching ...`, `Listening to ...` or `Competing in ...` activities.

**Reply by Fusity** [reply]
Ok thank you for reply, how i can change it ? `Playing ...` `Streaming ...` `Watching ...` ?

**Reply by AbstractUmbra** [reply]
You can use [`discord.Activity()`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Activity) instead of `discord.Game` for the presence. It takes a [`discord.ActivityType`](https://discordpy.readthedocs.io/en/latest/api.html#discord.ActivityType) enum as the first argument:

```py
import discord

# competing, for instance
discord.Activity(type=discord.ActivityType.competing, name="a battle")
```

**Comment by Fusity** [comment]
Ok thank you so much !
But in discordpy.readthedocs.io, i can see CustomActivity, what is it for ?

**Reply by AbstractUmbra** [reply]
This is for the ability to *read* CustomActivity activities from Members.
